pkg/kn/plugin/manager.go

```diff
@@ -165,18 +179,34 @@ func (manager *Manager) ListPluginsForCommandGroup(commandGroupParts []string) (
 	return plugins, nil
 }
 
-func isPartOfCommandGroup(commandGroupParts []string, name string) bool {
+func filterPluginsByCommandGroup(plugins PluginList, commandGroupParts []string) PluginList {
+	ret := PluginList{}
+	for _, pl := range plugins {
+		if isPartOfCommandGroup(commandGroupParts, pl.CommandParts()) {
+			ret = append(ret, pl)
+		}
+	}
+	return ret
+}
+
+func isPartOfCommandGroup(commandGroupParts []string, commandParts []string) bool {
+	if len(commandParts) != len(commandGroupParts)+1 {
+		return false
+	}
+	for i := range commandGroupParts {
+		if commandParts[i] != commandGroupParts[i] {
+			return false
+		}
+	}
```

Comment:
I can comment all these lines 193-200 so that function returns true always and all tests still pass...

Reply:
fixed